tests/integration/Microsoft.Bot.ApplicationInsights.Core.Tests/ServiceResolutionTests.cs

```diff
@@ -135,38 +136,47 @@ public void ServiceResolution_InvalidInstance()
         }
 
         /// <summary>
-        /// Prepare appsettings.json for test
+        /// Prepare appsettings.json for test.
         /// </summary>
         /// <remarks>Ensures appsettings.json file is set up (copy based on different sample files,
-        /// post-pended with a version.)  ie, appsettings.json.no_app_insights </remarks>
+        /// post-pended with a version.)  ie, appsettings.json.no_app_insights. </remarks>
         /// <param name="version">Post-pended onto the file name to copy (ie, "no_app_insights"). If null, put no file.</param>
         public void ArrangeAppSettings(string version = "default")
         {
-            try { File.Delete("appsettings.json"); }
-            catch { }
-            
+            try
+            {
+                File.Delete("appsettings.json");
+            }
+            catch
```

Comment:
I think that this try/catch block should be removed as nothing is caught here. Instead, the block should be used in the caller methods. @JuanAr, what are your thoughts on this?

Reply:
Though it would be nice, it might be outside the scope of StyleCop fixing 🤔

Reply:
Let's capture it and place it in another PR if this is like really an obvious and smelly code.
As usual, is this a well documented anti-pattern? Will it be spotted by some automatic tools like Code Analysis?

Reply:
This is some kind of delete-if-exists pattern.